docs/config.md

```diff
@@ -0,0 +1,104 @@
+`org.wso2.carbon.identity.application.authenticator.hypr.common` → `target`.
+- Copy the `org.wso2.carbon.identity.application.authenticator.hypr.common-1.0.0-SNAPSHOT.jar` file.
+- Navigate to `<IS_HOME>/repository/components/lib` directory and paste the `.jar` file.
+
+**Step 3:** Deploying the HYPR REST API
+- Navigate to `identity-outbound-auth-hypr/components` → `org.wso2.carbon.identity.application.authenticator.hypr.rest` 
+- →`org.wso2.carbon.identity.application.authenticator.hypr.rest.dispatcher`→ `target`.
+- Copy the `api#hypr.war` file.
+- Navigate to `<IS_HOME>/repository/deployment/server/webapps`.
+- Paste the `.war` file into the webapps directory.
+- Next navigate to `<IS_HOME>/repository/conf`.
+- Open `deployment.toml` file.
+- Add the following lines of codes.
+```toml
+[[resource.access_control]]
+context = "(.*)/api/hypr/v1/authentication/status/(.*)"
+secure = "false"
+http_method = "GET"
+
+[tenant_context]
+enable_tenant_qualified_urls = "true"
+enable_tenanted_sessions = "true"
+rewrite.custom_webapps=["/api/hypr/"]
+```
+
+**Step 4:** Deploying HYPR login Page
+- Copy `hyprlogin.jsp` in the downloaded artifacts.
+- Navigate to `<IS_HOME>/repository/deployment/server/webapps` → `authenticationendpoint`.
+- Paste or drop the `JSP` file in the `authenticationendpoint` directory.
+
+
+### _The WSO2 Console’s UI for HYPR authenticator section as follows_
+![Configuring HYPR in WSO2 Console](images/wso2Console.png)
+
+#### Base URL
+This refers to the Base URL  you received from the HYPR upon creating a tenant for your organization.
+Example :
+```
+https://<organization name>.hypr.com
+```
+
+#### Relying Party App ID
+This refers to the App ID you received for the application you created in the HYPR Control Center.
+Example :
+```
+<Application ID of the HYPR App>
+```
+Follow the following steps to extract the App ID From the HYPR Control Center.
+- Navigate to the HYPR Control Central via the link provided to you from HYPR and log in using your HYPR credentials.
+- Select your app from the `Choose an App` drop-down list. Next you are navigated to the Dashboard of the application. 
+- Finally, click the App Settings in the top right corner. Note down the App ID.
+
+#### API Token
+This refers to an API token newly generated specifically for the HYPR App via the control center.
+Example :
+```
+<Generated API Token for the HYPR App>
+```
+Follow the following steps to generate a new API token From the HYPR Control Center.
+- Navigate to the HYPR Control Central via the link provided to you from HYPR and log in using your HYPR credentials.
+- Select your app from the `Choose an App` drop-down list. Next you are navigated to the Dashboard of the application.
+- Under `Advanced Config` click Access Token.
+- Then click on `Generate Token`.
+- Finally, add a new name to your API token,  select API Access Token from selective options provided and click on `Create Token`.
+
+### _The HYPR authenticator's flow as follows_
```

Comment:
Let's use a different title for the heading.
And use a leading sentence before the image.
Any reason to use a different formatting for these headings?

Reply:
No specific reason for the different formatting. Will change it.

Reply:
Resolves via [3ef7a2c](https://github.com/wso2-extensions/identity-outbound-auth-hypr/pull/2/commits/3ef7a2caa1aafeb0302cc6662760cbe471bd299d).

Reply:
`And use a leading sentence before the image.` -> Resolves via [de6c3f2](https://github.com/wso2-extensions/identity-outbound-auth-hypr/pull/2/commits/de6c3f2ffe86e37ea70f323db0bdb540482ec1d8)